cs/ajax.texy

```diff
@@ -13,6 +13,9 @@ AJAXový požadavek lze detekovat metodou služby [zapouzdřující HTTP požada
 
 AJAXový požadavek se nijak neliší od klasického požadavku - je zavolán presenter s určitým view a parametry. Je také věcí presenteru, jak bude na něj reagovat: může použít vlastní rutinu, která vrátí nějaký fragment HTML kódu (HTML snippet), XML dokument, JSON objekt nebo kód v JavaScriptu.
 
+.[caution]
+**Upozornění:** návod neobsahuje implementaci AJAXových požadavků na straně klienta. Pro kompletní funkčnost uváděných příkladů doporučujeme nainstalovat knihovnu [nette.ajax.js | http://addons.nette.org/vojtech-dobes/nette-ajax-js] od Vojtěcha Dobeše.
```

Comment:
Nepsal bych to takhle negativně, informace že se prostě musí stáhnout ta knihovna ručně úplně stačí :)